docs/source/host_challenge.md

```diff
@@ -162,6 +162,44 @@ Finally run the `./run.sh` script in the bundle. It will generate a `challenge_c
 
 **Congratulations!** you have submitted your challenge configuration for review and [EvalAI team](https://eval.ai/team) has notified about this. [EvalAI team](https://eval.ai/team) will review and will approve the challenge.
 
+## Host a remote evaluation challenge
+
+### Step 1: Set up the challenge
+
+Follow [host challenge using github section](host_challenge.html#host-challenge-using-github) to set up a challenge on EvalAI.
+
+### Step 2: Edit challenge configuration
+
+Set the `remote_evaluation` parameter to `True` in [`challenge_config.yaml`](https://github.com/Cloud-CV/EvalAI-Starters/blob/621f0cb37b2f1951613c9b6c967ce35be55d34c8/challenge_config.yaml#L12). This challenge config file defines all the different settings of your challenge such as start date, end date, number of phases, and submission limits etc.
+
+Edit this file based on your requirement. For reference to the fields, refer to the [challenge configuration reference section](configuration.html).
+
+Please ensure the following fields are set to the following values:
+
+- `remote_evaluation : True`
```

Comment:
Can you also add one line after this saying

`Refer to the [following documentation](https://evalai.readthedocs.io/en/latest/configuration.html) for details on challenge configuration.`